components/brave_sync/brave_profile_sync_service_impl.cc

```diff
@@ -200,11 +200,22 @@ BraveProfileSyncServiceImpl::BraveProfileSyncServiceImpl(Profile* profile,
                  base::Unretained(this)));
 
   model_ = BookmarkModelFactory::GetForBrowserContext(profile);
+  // model_ can be null in some tests
 
   network_connection_tracker_->AddNetworkConnectionObserver(this);
   RecordSyncStateP3A();
 }
 
+void BraveProfileSyncServiceImpl::BookmarkModelLoaded(BookmarkModel* model,
+                                                      bool ids_reassigned) {
+  DCHECK(model);
```

Comment:
are these DCHECKs really necessary? `BookmarkModelLoaded` only gets called when the bookmark model is loaded so this would actually be debugging internal chromium code which doesn't seem necessary here

Reply:
>are these DCHECKs really necessary?
no that is my paranoia, if can't be null, but if it would be null, than all is totally wrong

Reply:
Fixed at https://github.com/brave/brave-core/pull/4588/files#diff-ab344835b09fb6ac1c601af787b116bbR211